Dockerfile

```diff
@@ -1,6 +1,12 @@
-FROM python:3.7
+FROM python:3.7-alpine AS requirements
+ADD requirements.txt .
+RUN apk add --no-cache libffi-dev gcc zlib-dev jpeg-dev freetype-dev libwebp libwebp-dev musl-dev
+RUN pip install -r requirements.txt
 
+FROM python:3.7-alpine
+RUN apk add --no-cache jpeg-dev libwebp libwebp-dev
 WORKDIR /app
-ADD requirements.txt /app
-
-RUN pip install -r requirements.txt
+COPY --from=requirements /usr/local/lib/python3.7/site-packages /usr/local/lib/python3.7/site-packages
+RUN adduser -D -h /app -s /bin/sh -u 1000 blurple
```

Comment:
Using a different user here makes it difficult to add additional dependencies for testing during development.